concepts/05 Widgets/DataGrid/00 Getting Started with DataGrid/35 Group Data.md

```diff
@@ -0,0 +1,109 @@
+Grid records can be grouped by values of one or multiple columns. To group records, users should drag and drop column headers onto the [groupPanel](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/groupPanel/). Enable the **groupPanel**.[visible](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/groupPanel/#visible) option to display this UI element.
+
+You can also group records programmatically. Specify the [groupIndex](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#groupIndex) for the columns by which records should be grouped. In this tutorial, the groupIndex is specified for the `Country` column:
```

Comment:
I see two possible issues here:
1. Specify the groupIndex **option** for ...
2. Doesn't this option requires a single column?

Reply:
1. I used "groupIndex" as plain text - "specify the group index". But I will add "option" because since we have multiple columns, it should have been "specify group indexes".
2. Not necessarily. You can specify group indexes for multiple columns to create groups inside groups.

```suggestion
You can also group records programmatically. Specify the [groupIndex](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#groupIndex) option for the columns by which records should be grouped. In this tutorial, the **groupIndex** is specified for the `Country` column:
```

Reply:
LGTM